system/HTTP/CURLRequest.php

```diff
@@ -651,11 +651,12 @@ protected function setCURLOptions(array $curlOptions = [], array $config = [])
 
         // version
         if (! empty($config['version'])) {
-            if ($config['version'] === 1.0) {
+            $version = sprintf('%.1f', $config['version']);
```

Comment:
The format is wrong. `f` is locale aware. See https://www.php.net/manual/en/function.sprintf.php#refsect1-function.sprintf-parameters

```suggestion
            $version = sprintf('%.1F, $config['version']);
```

Reply:
You are correct! Fixed and pushed a new update.